src/module/cp/oierdb.py

```
@@ -0,0 +1,370 @@
+def calculate_current_status(
+    first_award_year: int,
+    first_award_grade: str,
+    current_year: Optional[int] = None
+) -> str:
+    """
+    根据第一个奖项的年份和年级，推算选手在当前年份的状态
+    
+    Args:
+        first_award_year: 第一个奖项的年份
+        first_award_grade: 第一个奖项时的年级
+        current_year: 当前年份，默认2025
```

Ruff (0.13.3)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)